driver/main.c

```diff
@@ -1801,6 +1821,8 @@ TRACEPOINT_PROBE(syscall_enter_probe, struct pt_regs *regs, long id)
 	}
 #endif
 
+	this_cpu_inc(g_n_tracepoint_hit);
```

Comment:
For completeness, since these counters are exported via scap, does it make sense to increment the counter for page faults as well?